pyramid_tm/__init__.py

```diff
@@ -126,4 +136,18 @@ def includeme(config):
     - If none of the above conditions are True, the transaction will be
       committed (via ``transaction.commit()``).
     """
+    config.add_request_method(
+        'pyramid_tm.create_tm', name='transaction', reify=True,
```

Comment:
I passed the string instead of the function itself because it's what ``config.add_tween`` did below so I figured consistency was better.